.github/workflows/tag-on-merge.yml

```
@@ -0,0 +1,31 @@
+name: Tag on merge to master
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Compute tag name
+      id: compute-tag
+      run: |
+        echo Computing next tag number
```

```suggestion
        echo "Computing next tag number"
```

Not sure if this is needed? I notice we use the `echo "..."` pattern 3 lines down?

---

I'm not totally sure of the difference here? But [the runs](https://github.com/ben/progit2-test/runs/2925691880?check_suite_focus=true#step:3:7) work okay.

---

It seems to work OK on your `progit-2-test` repository.

Let's leave it as-is then. 😄